lib/function.g

```diff
@@ -638,20 +638,17 @@ end);
 
 BIND_GLOBAL( "VIEW_STRING_OPERATION",    function ( op )
     local   class,  flags,  types,  catok,  repok,  propok,  seenprop,
-            t, res;
+            t, flafs, res;
     class := "Operation";
     if IS_IDENTICAL_OBJ(op,IS_OBJECT) then
         class := "Filter";
     elif IS_CONSTRUCTOR(op) then
         class := "Constructor";
     elif IsFilter(op) then
         class := "Filter";
-        flags := FLAGS_FILTER(op);
-        if flags <> false then
-            flags := TRUES_FLAGS(flags);
-        else
-            flags := [];
-        fi;
+        flafs := FLAGS_FILTER(op);
+        if flafs <> false then
+        flags := TRUES_FLAGS(FLAGS_FILTER(op));
```

Comment:
Perhaps you meant to use `flafs` here, too? Otherwise you could just as well get rid of it again.